### PR TITLE
Made productlist page

### PR DESCRIPTION
**Pull request by poojavishnoi**
You can view the product list by clicking on the product image.Also I will add meta tag later regarding author and description.

### CODE REVIEW

File: css/footer.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  }
+  .footer_container h3 {
+    margin-top: 1rem;
+  }
+
+  .footer_icons img {
+    width: 1rem;
+    margin: 0.5rem;
+  }
+
```
Comment (rishbitsnbytes) [inline css/footer.css]: Suggestion to remove extra spacings.

**Review by rishbitsnbytes — COMMENTED** [review]
Looks good to merger.

File: css/footer.css (hunk 1 above)
Comment (poojavishnoi) [reply css/footer.css]: okayy.

**Review by anandkumarcoder — APPROVED** [review]
Look good for merging

**Pull request merged**